### PR TITLE
Plyer getting hit animation and some logic

### PR DESCRIPTION
**Pull request by balint-szabox**
- The player does not take dmg yet

### CODE REVIEW

**Comment by balint-szabox** [comment]
Ahoy @Baromfy and @kssdm,
I recommend that you only look at the Player.cs file. The rest is not really understandable on Git. If you need help ask me.

File: Assets/Scripts/Player.cs [hunk 1]
```diff
@@ -6,6 +7,7 @@ public class Player : MonoBehaviour
     [SerializeField] float runSpeed = 10f;
     [SerializeField] float jumpSpeed = 15f;
     [SerializeField] float climingSpeed = 9f;
```
Comment (kssdm) [inline Assets/Scripts/Player.cs]: *should be clim**b**ingSpeed ?

Comment (kssdm) [reply Assets/Scripts/Player.cs]: I understand the concepts in Player.cs in a general way (which function does what), but some tools or solutions are completely new. Anyway, looks cool. :D

Comment (balint-szabox) [reply Assets/Scripts/Player.cs]: I have changed the name to climbingSpeed.

**Pull request merged**